CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Additional logging of historical query flow in `UNSAFE` builds.
 - Historical query system will re-request entries if the host fails to provide them within a fixed time.
 - Renamed `ccf::CodeDigest` to `ccf:pal::PlatformAttestationMeasurement` and `get_code_id()` to `get_measurement()` (#5063).
+- Recovery procedure:
+  - The `GET /gov/recovery_share` endpoint has been replace with `GET /gov/recovery_share/{member_id}`. This can be accessed without any authentication.
```

Comment:
```suggestion
  - The `GET /gov/recovery_share` endpoint has been replaced with `GET /gov/recovery_share/{member_id}`. This can be accessed without any authentication.
```